src/components/proofs/ProofList.tsx

```diff
@@ -29,8 +31,18 @@ const bottomWrapper = classnames(
 )
 
 export function ProofListSuspended() {
-  const [category, setCategory] = useState<CategoriesTitles>('NFTs')
+  const params = useUrlParams()
+
+  const [category, setCategory] = useState<CategoriesTitles>(
```

Comment:
Let's make a parameter where a specific section will be specified, without such checks